src/librustdoc/core.rs

```diff
@@ -326,10 +328,6 @@ pub(crate) fn run_global_ctxt(
     // and might break if queries change their assumptions in the future.
 
     // NOTE: This is copy/pasted from typeck/lib.rs and should be kept in sync with those changes.
```

Comment:
Can you add a note how it's diverging from typeck here (and also update the typeck crate name :D)